src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export class Tabs extends React.Component<Props, {}> {
+  static findSelectedTab = (tabs: Tab[], selectedTabId: string) => {
+    return tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+  };
+
+  handleTabSelected = (
+    event: React.MouseEvent<HTMLAnchorElement>,
+    selectedTabId: string,
+    tab: Tab,
+  ) => {
+    event.preventDefault();
+
+    if (selectedTabId !== tab.id) {
+      this.props.onTabSelected(tab);
+    }
+  };
+
+  render() {
+    const { tabs, selectedTabId } = this.props;
+
+    return (
+      <>
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map(tab => (
+              <li
+                key={tab.id}
+                className={classNames({ 'is-active': tab.id === selectedTabId })}
+              >
+                <a
```

Comment:
link do not suites here, make it a button

Reply:
But Bulma requires the following structure:
a tabs container
a <ul> HTML element
a list of <li> HTML element
<a> HTML anchor elements for each link

https://bulma.io/documentation/components/tabs/